pages.tr/android/screencap.md

```diff
@@ -0,0 +1,9 @@
+# screencap
+
+> Bir mobil ekranın ekran görüntüsünü alın.
+> Bu komut sadece `adb shell` üzerinden kullanılabilir.
+> Daha fazla bilgi için: <https://developer.android.com/studio/command-line/adb#screencap>.
+
+- Bir ekran görüntüsü al:
+
+`screencap {{path/to/file}}`
```

Comment:
```suggestion
`screencap {{dosya/yolu}}`
```
Translate placeholders/command arguments for better readability to native readers.